wagtail/admin/templates/wagtailadmin/shared/side_panels/includes/status/locale.html

```diff
@@ -8,15 +8,15 @@
         {% trans 'Locale: ' as screen_reader_title_prefix %}
     {% endif %}
 
-    {% if translations %}
+    {% if object.pk and translations %}
```

Comment:
This is because in snippets we still want to show the locale selector on the create page, so `translations` is not empty, but we don't want to show the "Available in x locales" string.